src/js/intl-tel-input.ts

```diff
@@ -15,9 +17,9 @@ interface IntlTelInputInterface {
   getCountryData: () => Country[];
   getInstance: (input: HTMLInputElement) => Iti | null;
   instances: { [key: string]: Iti };
-  loadUtils: (path: string) => Promise<unknown> | null;
-  startedLoadingAutoCountry?: boolean;
-  startedLoadingUtilsScript?: boolean;
+  loadUtils: (source: string|UtilsLoader) => Promise<unknown> | null;
+  startedLoadingAutoCountry: boolean;
+  startedLoadingUtilsScript: boolean;
```

Comment:
The previous type annotation wasn’t quite correct for these two state variables (they either do not exist at all or they are `true`, never `false`). The Jasmine test suite sometimes resets them to `false` instead of deleting them, and then some tests expect them to be false, leading them to pass or fail depending on what order they were run in.

I figured it was simpler just to ensure they are always present and always a boolean.